src/Dispatcher.php

```diff
@@ -367,5 +371,37 @@ private function processError($error)
         }
     }
 
+    /**
+     * @param $warnings
+     */
+    private function processWarnings($warnings)
+    {
+        $msgs = [
+            1 => 'DIC poplatnika v datove zprave se neshoduje s DIC v certifikatu',
+            2 => 'Chybny format DIC poverujiciho poplatnika',
+            3 => 'Chybna hodnota PKP',
+            4 => 'Datum a cas prijeti trzby je novejsi nez datum a cas prijeti zpravy',
+            5 => 'Datum a cas prijeti trzby je vyrazne v minulosti '
+        ];
+
+        //Reset last warnings
+        $this->lastWarnings = null;
```

Comment:
Pokud to má být pole, tak reset by se měl provádět spíše jako `$this->lastWarnings = array();`